Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -2056,10 +2410,142 @@ void Tileset::processQueue(
 
   for (LoadRecord& record : queue) {
     CESIUM_TRACE_USE_TRACK_SET(this->_loadingSlots);
-    record.pTile->loadContent();
+    // TODO: for implicit tilesets only create content request if content is
+    // available.
+    if (record.pTile->getState() == Tile::LoadState::Unloaded) {
+      record.pTile->loadContent(this->requestTileContent(*record.pTile));
+    } else {
+      record.pTile->loadContent();
```

Comment:
If `loadContent` doesn't go back to requesting its own content... then `loadContent` does two totally different things, depending on whether the `Tile` is in the `Unloaded` state to start. Now that those two cases are separated at the call site, it'd be a lot cleaner to make them two separate methods.

Reply:
I agree with this, I'll have to sort out what exactly happens in those two different cases, but I think that separation is worth making.